### PR TITLE
auto confirmation flags added to dockerfile

### PR DESCRIPTION
**Pull request by batikanu**
creating docker image fails, in case confirmation is required for updates and installs without the -y flag.
@taimir can you review?

### CODE REVIEW

**Comment by taimir** [comment]
Just a second, let me see.

File: cluster/images/hyperkube/Dockerfile [hunk 1]
```diff
@@ -66,7 +66,7 @@ ADD https://storage.googleapis.com/kubernetes-release/easy-rsa/easy-rsa.tar.gz /
 
 # Copy CNI binary and configuration files
 
-RUN apt-get update && apt-get upgrade && apt-get install xz-utils  && rm -rf /var/lib/apt/lists/*
+RUN apt-get update -y && apt-get upgrade -y && apt-get install -y xz-utils  && rm -rf /var/lib/apt/lists/*
```
Comment (taimir) [inline cluster/images/hyperkube/Dockerfile]: `apt-get update` is already called at line `21`:

```{sd}
RUN DEBIAN_FRONTEND=noninteractive apt-get update -y \
...
```

so we can probably skip it. Other than that this is fine. It's also an option to move these `apt-get` calls to the top of the file, where `apt-get` is currently called. Not sure if we want to mix in our changes with the previous state, though.

**Comment by taimir** [comment]
1 comment, LGTM when it's resolved

File: cluster/images/hyperkube/Dockerfile (hunk 1 above)
Comment (taimir) [inline cluster/images/hyperkube/Dockerfile]: When I look at this again, the `rm -rf /var/lib/apt/lists/*`. I'd suggest just moving `xz-utils` to the top of the file, and deleting this whole line ;)

**Comment by cheld** [comment]
Batikan is on vacation. Just merge and fix

**Pull request merged**